cmd/shipper-state-metrics/collector.go

```diff
@@ -322,6 +332,36 @@ func (ssm ShipperStateMetrics) collectClusters(ch chan<- prometheus.Metric) {
 	}
 }
 
+func (ssm ShipperStateMetrics) collectRolloutBlocks(ch chan<- prometheus.Metric) {
+	nss, err := getNamespaces(ssm.nssLister)
+	if err != nil {
+		glog.Errorf("collect Namespaces: %s", err)
+		return
+	}
+
+	rolloutBlocks, err := ssm.rbLister.List(everything)
+	if err != nil {
+		glog.Errorf("collect RolloutBlocks: %s", err)
+		return
+	}
+
+	rbsPerNamespace := make(map[string]float64)
+	for _, rolloutBlock := range rolloutBlocks {
+		rbsPerNamespace[rolloutBlock.Namespace]++
```

Comment:
Minor: if I understand correctly, float64 is chosen because prometheus gauge wants it. In the other hand, the semantics of this value is a counter, which means an integer-like value is more natural for operations like ++. See https://stackoverflow.com/questions/8561393/is-using-increment-operator-on-floats-bad-style for more details. As I said, this is a matter of polishing the semantics of this method, so, not a blocker to me.